Documents/UserManual/userManualFrench.md

```diff
@@ -0,0 +1,265 @@
+
+## Avertissements
+
+- <h3><img src="./Img/warning.png" width="20"/> Ne jamais démonter ou modifier, la modification et/ou le désassemblage du produit peut entraîner un choc électrique ou tout autre type de blessure.</h3>
+
+- <h3><img src="./Img/warning.png" width="20"/> Faire preuve d'une prudence particulière à l'égard des enfants et des animaux domestiques, notamment dans les foyers où ils sont présents.</h3>
+
+- <h3><img src="./Img/warning.png" width="20"/> Soyez prudent lorsque vous chargez l'appareil, vous risquez de provoquer un incendie ou un choc électrique. N'appliquez pas trop de chaleur, de pression ou de tension au cordon d'alimentation et aux câbles.</h3>
+
+- <h3><img src="./Img/warning.png" width="20"/> Ne fixez rien d'autre que les éléments spécifiés à l'appareil, toute négligence pourrait entraîner des blessures ou des dommages.</h3>
+
+<div style="page-break-after: always;"></div>
+
+# Premières Etapes
+
+SportShield est un dispositif antivol innovant conçu pour les équipements sportifs, équipé de capteurs et d'actionneurs qui détectent les mouvements, déclenchent une alarme et avertissent le propriétaire.
+<br><br>
+**SportShield est encore en cours de développement. Ce prototype n'inclut pas encore toutes les fonctionnalités que nous prévoyons d'implémenter.**
+
+## Contenu de la boîte
+
+Votre SportShield inclut:
+
+- Un dispositif antivol SportShield<br>
+<img src="./Img/deviceFrench.png" width="225"/>
+<img src="./Img/deviceBottom.png" width="200"/>
+- Un chargeur
+- Ce mode d'emploi
+
+## Chargez votre SportShield
+
+Si l'appareil SportShield est à court de batterie, il ne fonctionnera plus jusqu'à ce qu'il soit rechargé. <br>
+Un SportShield entièrement chargé peut durer plus de 6 jours.
+
+Pour charger SportShield:
+
+1. Branchez le chargeur dans le port USB sur le SportShield.<br>
+<img src="./Img/chargingPortFrench.png" width="200"/>
+2. Branchez l'autre bout du chargeur dans la prise.
+
+<br>
+
+SportShield est en train de charger. <br>
+**CAUTION:** Essayez de ne pas utiliser SportShield pendant qu'il charge.
+
+<div style="page-break-after: always;"></div>
+
+## Set up SportShield
+
+Pour utiliser pleinement le dispositif SportShield, vous aurez besoin de set up quelques points avec votre smartphone. <br>
+Vous pouvez le faire soit par Bluetooth, avec une carte NFC ou avec l'appli SportShield. <br><br>
+Par Bluetooth:
+
+- Activez votre connection Bluetooth sur votre smartphone.
+- Vous devrez trouver un dispositif nommé "SportShield".<br>
+<img src="./Img/findBluetooth.jpg" width="200"/>
+- Connectez votre smartphone avec le dispositif. <br>
+<img src="./Img/BluetoothSettings.png" width="200"/>
+<br><br>
+  
+Avec une carte NFC:
+
+- Faites toucher l'appareil NFC et le SportShield.
```

Comment:
Replace this sentence with "Rentrez la carte NFC en contact avec votre SportShield".